$MirOS: ports/lang/pcc/patches/patch-configure_ac,v 1.4 2011/04/02 13:15:01 tg Exp $

	• brand and anonymise
	• fix cross-compiling

--- configure.ac.orig	Fri Apr  1 13:58:07 2011
+++ configure.ac	Sat Apr  2 17:30:05 2011
@@ -361,7 +361,11 @@ else
         AC_MSG_RESULT([not cross compiling])
         CC_FOR_BUILD=${CC-cc}
         AC_SUBST(CC_FOR_BUILD)
+	test -n "$CPPFLAGS_FOR_BUILD" || CPPFLAGS_FOR_BUILD=$CPPFLAGS
+	test -n "$CFLAGS_FOR_BUILD" || CFLAGS_FOR_BUILD=$CFLAGS
 fi
+AC_SUBST(CPPFLAGS_FOR_BUILD)
+AC_SUBST(CFLAGS_FOR_BUILD)
 
 AC_CACHE_CHECK([for C99 printf size specifiers], ac_have_c99_format, [
   AC_RUN_IFELSE([
@@ -439,7 +443,7 @@ AC_SUBST(ADD_CPPFLAGS)
 pcc_major=`echo $PACKAGE_VERSION | awk -F. '{print $1}'`
 pcc_minor=`echo $PACKAGE_VERSION | awk -F. '{print $2}'`
 pcc_minorminor=`echo $PACKAGE_VERSION | awk -F. '{print $3}'`
-versstr="\"$PACKAGE_STRING `cat $srcdir/DATESTAMP` for $target, $USER@`hostname`\""
+versstr="\"$PACKAGE_STRING `cat $srcdir/DATESTAMP` for $target, ${MPVERSION-$USER@`hostname`}\""
 
 AC_DEFINE_UNQUOTED(TARGOS, $targos, [Target OS])
 AC_DEFINE_UNQUOTED(TARGOSVER, $targosver, [Target OS version])
